# Capstone Project - The Battle of Neighborhoods

## Preampble




#### Initial Directories

In [94]:
#pip install pandas

In [95]:
#conda install -c conda-forge geopy --yes

In [96]:
#pip install seaborn

In [97]:
#pip install earthpy

In [98]:
#pip install geojsonio

In [99]:
#pip install shapely

In [100]:
#pip install folium

In [101]:
#conda install --channel conda-forge geopandas

In [102]:
#pip install IPython

In [103]:
#pip install foursquare


In [11]:
#pip install -U googlemaps

In [12]:
#pip install geojson

In [104]:
#pip install python -m webbrowser -t

In [105]:
#pip install --upgrade pandas

In [107]:
#pip install beautifulsoup4

In [106]:
#pip install jsonquery

#### Import Modules

In [31]:
import pandas as pd
import numpy as np
import scipy as sp
import matplotlib.pyplot as plt
import seaborn as sns
import pylab as pl
import json
import folium
#import geopandas as gpd
#import earthpy as et
import IPython
import geojson
import foursquare as fs
#import beautifulsoup4 as bs4
#from bs4 import BeautifulSoup
#import geopandas as gpd

## Introduction/Business Problem



### Introduction where you discuss the business problem and who would be interested in this project.

Hospital Waiting Lists in Northern Ireland 



## Data

where you describe the data that will be used to solve the problem and the source of the data.

According to the question, I need to find following data to resolve the problem.

A. Geographic coordinate of Northern Ireland hospitals
I will need to find location data for Northern Ireland hospitals ... I can find data by carrying out a series of searchs using Foursquare Location Venues search on hospitals around a range of locations in Northern Ireland 

* Belfast Region 54.5973° N, 5.9301° W
* Lisburn Region 54.5162° N, 6.0580° W    
* Newtownabbey Region 54.6792° N, 5.9591° W
* Bangor Region 54.6542° N, 5.6731° W
* Antrim Region 54.7195° N, 6.2072
* Ballycastle Region 55.2047° N, 6.2532
* Coleraine Region 55.1326° N, 6.6646° W    
* Derry Region 54.9966° N, 7.3086° W
* Omagh Region 54.5977° N, 7.3100° W
* Enniskillen Region   54.3438° N, 7.6315° W 
* Newry Region 54.1751° N, 6.3402° W
* Armagh Region 54.3503° N, 6.6528° W
* Downpatrick Region 54.3288° N, 5.7157° W


Get Location Data from FOURSQUARE<sup>TM</sup>

In [70]:
# intialise data of lists.
locdata = {'Region':['Belfast', 'Lisburn', 'Newtonabbey', 'Bangor', 'Antrim', 'Ballycastle', 'Coleraine', 'Derry', 'Omagh', 'Eniskillen', 'Newry', 'Armagh', 'Downpatrick'], 'Latitude':[54.6, 54.5, 54.7, 54.7, 54.7, 55.2, 55.1, 55.0, 54.6, 54.3, 54.2, 54.4, 54.3],'Longitude':[-5.9, -6.0, -6.0, -5.7, -6.2, -6.3, -6.7, -7.3, -7.3, -7.6, -6.3, -6.7, -5.7]}
# intialise data of lists.
locdf = pd.DataFrame(locdata) 
# Print the output. 

print(locdf)

         Region  Latitude  Longitude
0       Belfast      54.6       -5.9
1       Lisburn      54.5       -6.0
2   Newtonabbey      54.7       -6.0
3        Bangor      54.7       -5.7
4        Antrim      54.7       -6.2
5   Ballycastle      55.2       -6.3
6     Coleraine      55.1       -6.7
7         Derry      55.0       -7.3
8         Omagh      54.6       -7.3
9    Eniskillen      54.3       -7.6
10        Newry      54.2       -6.3
11       Armagh      54.4       -6.7
12  Downpatrick      54.3       -5.7


In [110]:
#url2='https://api.foursquare.com/v2/venues/search?ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=100&v=20180628'

#Belfast Region
url1='https://api.foursquare.com/v2/venues/search?ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=1&v=20180628'

#Lisburn Region 54.5162° N, 6.0580° W    
url2='https://api.foursquare.com/v2/venues/search?ll=54.5,-6.06&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=20&v=20180628'

#Newtownabbey Region 54.6792° N, 5.9591° W
url3='https://api.foursquare.com/v2/venues/search?ll=54.67,-5.96&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=20&v=20180628'
 
#Bangor Region 54.6542° N, 5.6731° W
url4='https://api.foursquare.com/v2/venues/search?ll=54.65,-5.67&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=20&v=20180628'
    
#Antrim Region 54.7195° N, 6.2072
url5='https://api.foursquare.com/v2/venues/search?ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'

#Ballycastle Region 55.2047° N, 6.2532
url6='https://api.foursquare.com/v2/venues/search?ll=55.2,-6.25&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'

#Coleraine #55.1326° N, 6.6646° W
url7='https://api.foursquare.com/v2/venues/search?ll=55.13,-6.66&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=20&v=20180628'
    
#Derry Region    
url8='https://api.foursquare.com/v2/venues/search?ll=55.0,-7.3&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'

#Omagh Region
url9='https://api.foursquare.com/v2/venues/search?ll=54.6,-7.7&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'

#Enniskillen Region   54.3438° N, 7.6315° W 
url10='https://api.foursquare.com/v2/venues/search?ll=54.34,-7.6&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'

#Newry Region 54.1751° N, 6.3402° W
url11='https://api.foursquare.com/v2/venues/search?ll=54.17,-6.34&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628' 

#Armagh Region 54.3503° N, 6.6528° W
url12='https://api.foursquare.com/v2/venues/search?ll=54.3,-6.6&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'

#Downpatrick Region
url13='https://api.foursquare.com/v2/venues/search?ll=54.3,-6.6&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=50&v=20180628'


In [62]:
url2='https://api.foursquare.com/v2/venues/search?ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=100&v=20180628'

#count = 0
#while (count < 13):     
#    JSC = requests.get(url1).json()
#    print("Hello Geek") 
#    venues = JSC['response']['venues']

    
#Belfasth=requests.get(url1).json()
#Lisburnh=requests.get(url2).json()
#Newtonabbeyh=requests.get(url3).json()
#Bangorh=requests.get(url4).json()
#Antrimh=requests.get(url5).json()
#Ballycastleh=requests.get(url6).json()
#Coleraineh=requests.get(url7).json()
#Derryh=requests.get(url8).json()
#Omaghh=requests.get(url9).json()
#Eniskillenh=requests.get(url10).json()
#Newryh=requests.get(url11).json()
#Armaghh=requests.get(url12).json()
#Downpatrickh=requests.get(url13).json()
#JSONContent = requests.get(urll2).json()


#content = json.dumps(Belfasth, indent = 6, sort_keys=True)


#assign relevant part of JSON to venues
#venues = Belfasth['response']['venues']

# tranform venues into a dataframe
#dataframe = json_normalize(venues)
#dataframe.head()

#import webbrowser

#webbrowser.open(url)
#results = requests.get(url).json()

#https://api.foursquare.com/v2/venues/search?ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=100&v=20180628

    
###venues = results['response']
#json_normalize(venues)
#ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=100&v=20180628


#print("All Data Imported")

In [112]:

import pandas as pd
#count = 1
#while (count < 13):
#    file='url'+str(count)
#    print(file)


#results = requests.get(url1).json()
A = requests.get(url13).json()
venues = A['response']['venues']
    
dataframe = pandas.json_normalize(venues)
dataframe.head()

    
#assign relevant part of JSON to venues

#venues = JSC['response']['venues']

dataframe = pandas.json_normalize (venues)
dataframe.head()

# keep only columns that include venue name, and anything that is associated with location
filtered_columns = ['name', 'categories'] + [col for col in dataframe.columns if col.startswith('location.')] + ['id']
dataframe_filtered = dataframe.loc[:, filtered_columns]


# function that extracts the category of the venue
def get_category_type(row):
    try:
        categories_list = row['categories']
    except:
        categories_list = row['venue.categories']
        
    if len(categories_list) == 0:
            return None
    else:
         return categories_list[0]['name']

# filter the category for each row
dataframe_filtered['categories'] = dataframe_filtered.apply(get_category_type, axis=1)

# clean column names by keeping only last term
dataframe_filtered.columns = [column.split('.')[-1] for column in dataframe_filtered.columns]

dataframe_filtered2=dataframe_filtered.drop(columns=['labeledLatLngs', 'distance','postalCode','cc','state','crossStreet','address','id','formattedAddress'])

dataframe_filtered2

name     categories        lat  \
0                     Craigavon Area Hospital       Hospital  54.434523   
1                          St Luke's Hospital       Hospital  54.363271   
2                       South Tyrone Hospital       Hospital  54.522772   
3             Podiatry Dept., Lurgan Hospital       Hospital  54.457130   
4                         Daisy Hill Hospital       Hospital  54.179651   
5                      A&E Craigavon Hospital       Hospital  54.434674   
6                         Day Clinical Centre       Hospital  54.179270   
7                         CAH Ramone Building       Hospital  54.434472   
8                   Armagh Community Hospital       Hospital  54.350679   
9   Day Surgery Unit, Craigavon Area Hospital       Hospital  54.434287   
10                  CAH Elective Surgery Ward  Hospital Ward  54.434410   
11                        St Davnet's Complex       Hospital  54.248779   
12                         St Mary's Hospital       Hospital  54.120330   
13                            Lurgan Hospital       Hospital  54.457198   
14                Mandeville Cancer Care Unit       Hospital  54.433899   
15                      South Tyrone Hospital       Hospital  54.510156   
16                   Markethill Health Centre       Hospital  54.296870   
17                  Monaghan General Hospital       Hospital  54.249337   

         lng         city         country  
0  -6.409912    Craigavon  United Kingdom  
1  -6.652340          NaN  United Kingdom  
2  -6.789000    Dungannon  United Kingdom  
3  -6.338271       Lurgan  United Kingdom  
4  -6.350720        Newry  United Kingdom  
5  -6.409710    Craigavon  United Kingdom  
6  -6.350635        Newry  United Kingdom  
7  -6.409973    Craigavon  United Kingdom  
8  -6.642694          NaN  United Kingdom  
9  -6.406690    Craigavon  United Kingdom  
10 -6.406862          NaN  United Kingdom  
11 -6.968883     Monaghan         Ireland  
12 -6.745119  Co Monaghan         Ireland  
13 -6.338170          NaN  United Kingdom  
14 -6.407751          NaN  United Kingdom  
15 -6.768836    Dungannon  United Kingdom  
16 -6.519070          NaN  United Kingdom  
17 -6.974511          NaN         Ireland

In [9]:
CLIENT_ID = E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN
CLIENT_SECRET = 5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402
VERSION = 20180628
LIMIT =50


url='https://api.foursquare.com/v2/venues/explore?ll=54.6,-5.9&categoryId=4bf58dd8d48988d196941735&client_id=E1DFI4SYQLTFLMNIUXB2BYBAT1WZFR3FIGIUP1QECWCAMVPN&client_secret=5J1XJPE51YXN3NORX1TAUOY53DXOUW0GCSQDMWDR3VMVZ402&limit=100&v=20180628

SyntaxError: invalid syntax (<ipython-input-9-0575d7736ef7>, line 2)

## Data Sources for Classification and Analysis

Data for this exercise is extracted from https://www.opendatani.gov.uk/group/health <br>
Emergency Care Waiting Times https://www.opendatani.gov.uk/dataset/emergency-care-waiting-times<br>
Outpatient Waiting Times https://www.opendatani.gov.uk/dataset/outpatient-waiting-times<br>
Inpatient Waiting Times https://www.opendatani.gov.uk/dataset/inpatient-waiting-times<br>
Diagnostic Waiting Times https://www.opendatani.gov.uk/dataset/diagnostic-waiting-times<br>
Cancer Waiting Times https://www.opendatani.gov.uk/dataset/cancer-waiting-times<br>

 Contains public sector information licensed under the Open Government Licence v3.0.

In [108]:
EmergWL = pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-ecwt-tables-q1-18-19.csv")

FirstCOPWL = pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-outpatients-q2-19-20.csv")

CentralCOPWL = pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-outpatients-centralisation-q2-19-20.csv")
ICATSWL=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-icats-q2-19-20.csv")
DayCaseIPWL=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-day-cases-q2-19-20.csv")
AdminIPWL=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-inpatients-q2-19-20.csv")
TotalIPWL=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-total-waiting-q2-19-20.csv")

DiagnosticWT=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-diagnostic-waiting-times-q2-19-20.csv")
DiagnosticRTT=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-tables-diagnostic-reporting-turnaround-q2-19-20.csv")

HSCT62DCWT=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-cwt-62-day-wait-by-trust-q1-19-20.csv")
TS62DCWT=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-cwt-62-day-wait-by-tumour-q1-19-20.csv")
HSCT31DCWT=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-cwt-31-day-wait-by-trust-q1-19-20.csv")
TS62DCWT=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-cwt-31-day-wait-by-tumour-q1-19-20.csv")  
CWT14D=pd.read_csv(r"https://www.health-ni.gov.uk/sites/default/files/publications/health/hs-niwts-cwt-14-day-wait-q1-19-20.csv")
TBMpath="https://www.opendatani.gov.uk/dataset/0b04b46c-49af-45d5-b277-91b10937a01b/resource/645f8eef-8813-47a9-bb1e-a4932ada721a/download/trustboundaries.geojson"

print("All Data Imported")

All Data Imported


In [109]:
TrustMap = folium.Map(location=[54.6, -7.7], tiles = "Stamen Terrain", zoom_start=8.45)
#,
TrustMap


In [ ]:
from IPython.display import GeoJSON

GeoJSON({
    "type": "Feature",
    "geometry": {
        "type": "Point",
        "coordinates": [54.6, -7.7]
    }
})
#<iframe width="700" height="400" src="https://www.opendatani.gov.uk/dataset/department-of-health-trust-boundaries/resource/645f8eef-8813-47a9-bb1e-a4932ada721a/view/eaa70ef4-6e10-492c-a464-e6c6769b0fb8" frameBorder="0"></iframe>

In [ ]:
IPython.display.JSON(data=None, url=None, filename=None)

In [ ]:
import json

with open('https://www.opendatani.gov.uk/dataset/0b04b46c-49af-45d5-b277-91b10937a01b/resource/645f8eef-8813-47a9-bb1e-a4932ada721a/download/trustboundaries.geojson') as f:
    data = json.load(f)

for feature in data['features']:
    print (feature['geometry']['type'])
    print (feature['geometry']['coordinates'])
    
#TrustMap.choropleth(geo_path='TBMpath')
#####TrustMap


In [ ]:
CentralCOPWL.head()

In [ ]:
CentralCOPWL.head()
data.CentralCOPWL = pd.to_numeric(data.CentralCOPWL, errors='coerce').fillna(0).astype(int)
print (data)

# Making a list of missing value types
#mdf = CentralCOPWL.replace(''--'',0)
#mdf.head()

In [ ]:
#Kmeans Cluster

from sklearn.cluster import KMeans
kmeans = KMeans(n_clusters=4)
kmeans.fit(X)
y_kmeans = kmeans.predict(X)

## RESULTS
- Insert Results here
Observations & Recommendations

# Appendix 1-  Data definitions

### Inpatient and Day Case Activity

#### Inpatient Admission
Inpatient admissions include both (a) patients admitted electively with the expectation that they will remain in hospital for at least one night, and (b) non-elective admissions (e.g. emergency admissions). A patient who is admitted with this intention but who leaves hospital for any reason without staying overnight is still counted as an inpatient. Day cases and regular attenders are not included.<br>

#### Day Case
A patient admitted electively during the course of a day with the intention of receiving care who does not require the use of a hospital bed overnight and who returns home as scheduled. If this original intention is not fulfilled and the patient stays overnight, such a patient should be counted as an inpatient.  Regular attenders have been included in day case figures with the exception of the acute programme of care.<br>

#### Average Available/Occupied Beds
The average number of available and occupied beds during the year in wards that are open overnight, measured at midnight. Hospitals may also have a number of beds in wards that are only open during the day. Beds reserved for day care admission or regular day admission are not included.<br>

### Outpatient Activity 

#### Outpatient
A patient who attends a clinic to see a consultant, a member of their team, or a locum for such a member.<br>

#### Outpatient Did Not Attend – DNA
DNA refers to the number of patients with an appointment who did not attend and failed to give advance warning to the hospital. This includes patients who cancelled their outpatient appointment on the same day on which the appointment was scheduled. The DNA rate is calculated by taking the number of missed appointments (DNAs) as a rate of the sum of the total number of attendances and missed appointments. <br>

#### Appointment cancelled by patient – CNA
A patient cancellation refers to an appointment that was intended to be held but was cancelled by the patient, with the patient contacting the hospital no later than the day before the appointment is scheduled and informing the hospital that they are unable to attend the scheduled appointment. The CNA rate is calculated by taking the number of patient cancellations (CNAs) as a rate of the sum of the total number of attendances and patient cancellations.<br>

#### Appointment cancelled by the hospital
An appointment that was intended to be held, but which did not occur, due to circumstances within the hospital. Appointments may be cancelled by the hospital for a variety of reasons. These include, in order to reschedule the appointment to an alternative date and due to the unavailability of the consultant. The hospital cancellation rate is calculated by taking the number of appointments cancelled by hospitals as a rate of the sum of the total number of attendances and appointments cancelled by hospitals.<br>

Note: The number of attendances does not equate to the number of individual patients seen, as it is possible for the same person to attend a consultant led outpatient service more than once during the year. This is the same when looking at missed and cancelled appointments, as it is possible for the same person to miss or cancel their appointment or have their appointment cancelled by the hospital more than once during the year.<br>
### Regional Assessment and Surgical Centres (RASC)
In October 2016 the then Health Minister launched ‘Health and Wellbeing 2026: Delivering Together,’ a strategy which underpins the Northern Ireland Executive’s draft Programme for Government ambition to support people to lead long, healthy and active <br>lives.
As part of this strategy, the Elective Care Plan was published in February 2017, which stated that ‘Regional Elective Care Assessment and Treatment Centres will be established to deliver large volumes of assessments and non-complex routine surgery across a broad range of specialties.’ In February 2019, prototype ‘Regional Assessment and Surgical Centres’ became operational for the surgical treatment of Cataracts. Patients waiting for these procedures can now be referred to a RASC for treatment rather than attend the hospital site they may have been referred to previously.<br>

### Integrated Clinical Assessment and Treatment Services (ICATS)

ICATS is the term used for a range of outpatient services for patients, which are provided by integrated multi-disciplinary teams of health service professionals, including GPs with a special interest, specialist nurses and allied health professionals. They are provided in a variety of primary, community and secondary care settings and they include assessment, treatment, diagnostic and advisory services.<br>

### ICATS Appointment<br>
An ICATS Tier 2 appointment is a non consultant outpatient appointment. Following ICATS Triage, patients who have not been given either a discharge, advice only or referral incomplete outcome will proceed for either a (i) first outpatient appointment, (ii) a diagnostic test or (iii) an ICATS Tier 2 appointment. Following a first Tier 2 ICATS appointment there are a number of outcomes, including a review Tier 2 ICATS appointment or a referral for a first consultant led outpatient appointment. <br>

ICATS Tier 2 appointments attended, those missed by patients (DNA), cancelled by patients (CNA) or cancelled by the hospital (Hospital Cancellations) are defined similarly to those for consultant led outpatient services. Reasons for cancellation of ICATS Tier 2 appointments are not currently collected by the Department.<br>

### Virtual Activity<br>
A virtual outpatient appointment is a planned contact by a healthcare professional responsible for the care of a patient for the purposes of clinical consultation, advice and treatment planning. Virtual appointments may take the form of a telephone contact, video-link intervention, an email or a letter.<br>

This information was collated by Hospital Information Branch, DoH.<br>

Further information is available from:<br>

Hospital Information Branch<br>
Department of Health<br>
Annex 2, Castle Buildings<br>
Stormont, BT4 3SQ<br>
Telephone: 028 90 522800<br>
E-mail:  statistics@health-ni.gov.uk<br>

5. For media enquiries please contact the DoH Press Office 028 9052 0575 or email pressoffice@health-ni.gov.uk. Out-of-hours please contact the Duty Press Officer on 028 9037 8110.

